src/os_install.c

```diff
@@ -53,7 +53,7 @@ static void print_help(void)
 	print("Options:\n");
 	print("   -V, --version=[VER]         If the version to install is not the latest, it can be specified with this option\n");
 	print("   -x, --force                 Attempt to proceed even if non-critical errors found\n");
-	print("   -B, --bundles=[BUNDLES]     Include the specified BUNDLES in the OS installation. Example: --bundles=os-core,vi\n");
+	print("   -B, --bundles=[BUNDLES]     Include the specified BUNDLES in the OS installation. Example: --bundles=os-core,vim\n");
```

Comment:
I just thought one more thing, many we should also remove os-core from that list, change it with `openssl` or something like that since `os-core` is installed by default, this may confuse people.

Reply:
yes. it's a good idea. I am going to add joe, because it's small. It's a good bundle for testing